src/main/java/Parser/Parser.java

```diff
@@ -0,0 +1,29 @@
+package Parser;
+
+import Command.*;
```

Comment:
Should wildcard imports be used? Maybe it is better to list them out?

Reply:
I think * is better as I am importing almost every command in that package

Reply:
HMMM actually now that I saw this " Rationale: Importing classes explicitly gives an excellent documentation value for the class at hand and makes the class easier to comprehend and maintain. Appropriate tools should be used in order to always keep the import list minimal and up to date. IDE's can be configured to do this easily." in the style guide, I am feeling conflicted.

Reply:
How can IDE be configured to do it though?